### PR TITLE
Feat(delete-exposure)

### PR DESCRIPTION
**Pull request by Noam6421**
This PR implements features: [614](https://dev.azure.com/spectrumFactory/CoronaI/_workitems/edit/614/), [1479](https://dev.azure.com/spectrumFactory/CoronaI/_workitems/edit/1479/).

#### Major changes: 
* Added trash icon to each exposure or flight when clicked give alert if confirmed deletes the exposure/ flight.
* Added new route to delete exposures from db: '/exposure/deleteExposure'.

### CODE REVIEW

File: client/src/components/App/Content/InvestigationForm/TabManagement/ExposuresAndFlights/useExposuresAndFlights.ts [hunk 1]
```diff
@@ -129,6 +124,40 @@ export const useExposuresAndFlights = (props : Props) => {
...
+            cancelButtonText: 'בטל',
+            cancelButtonColor: theme.palette.error.main,
+            confirmButtonColor: theme.palette.primary.main,
+            confirmButtonText: 'כן, המשך',
+        }).then((result) => {
+            if (result.value) {
+                deletingExposureLogger.info('launching exposure delete request', Severity.LOW);
+                setIsLoading(true);
+                if (updatedExpousres[index].id){
+                    const exposureId = updatedExpousres[index].id;
```
Comment (y-golde) [inline client/src/components/App/Content/InvestigationForm/TabManagement/ExposuresAndFlights/useExposuresAndFlights.ts:141]: moved updatedExposures[index] to a const, same for line 130 or do something like `const {wasAbroad , id} = upadtedExposures[index]`

File: server/src/ClientToDBAPI/ExposureRoute/mainRoute.ts [hunk 2]
```diff
@@ -160,4 +160,25 @@ exposureRoute.post('/updateExposures',handleInvestigationRequest, (request: Requ
         })
 });
 
+exposureRoute.delete('/deleteExposure',handleInvestigationRequest, (request: Request, response: Response) => {
+    const deleteExposureLogger = logger.setup({
+        workflow: `deleting investigation's exposure by id`,
+        user: response.locals.user.id,
+        investigation: response.locals.epidemiologynumber
+    });
+    console.log(request.query)
```
Comment (y-golde) [inline server/src/ClientToDBAPI/ExposureRoute/mainRoute.ts:169]: remove log

**Review by y-golde — CHANGES_REQUESTED** [review]

File: client/src/components/App/Content/InvestigationForm/TabManagement/ExposuresAndFlights/useExposuresAndFlights.ts (hunk 1 above)
Comment (Noam6421) [reply client/src/components/App/Content/InvestigationForm/TabManagement/ExposuresAndFlights/useExposuresAndFlights.ts:141]: 👍🏼

File: server/src/ClientToDBAPI/ExposureRoute/mainRoute.ts (hunk 2 above)
Comment (Noam6421) [reply server/src/ClientToDBAPI/ExposureRoute/mainRoute.ts:169]: 👍🏼

**Review by y-golde — APPROVED** [review]

**Review by barnahum8 — APPROVED** [review]

**Pull request merged**